docs/introduction/reducers.md

```diff
@@ -20,26 +17,33 @@ For complex applications, it is often useful to break down the global `reducer`
 ```lua
 local friendsReducer = function(state, action)
 	--[[
-		The state will be nil the first time this reducer is executed.
+		The state might be nil the first time this reducer is executed.
 		In that case, we need to initialize our state to be the empty table.
 	]]
 	state = state or {}
 
-	if action.type == "MadeANewFriend" then
+	if action.type == "MadeNewFriends" then
 		local newState = {}
 
 		-- Since state is read-only, we copy it into newState
 		for index, friend in ipairs(state) do
 			newState[index] = friend
 		end
 
-		table.insert(newState, action.friend)
+		for _, friend in ipairs(action.newFriends)
+			table.insert(newState, friend)
+		end
+
 		return newState
 	end
 
 	return state
 end
 
+--[[
+	note that the reducer for our entire application is defined by a table of sub-reducers where each sub-reducer is responsible for one portion of the
+	overall state.
+]]
```

Comment:
Formatting? Line 44 looks way too long, and should also probably start with a capital letter.